src/config/dynamic-theme-fixes.config

```diff
@@ -19598,8 +19598,20 @@ img[src*="/img/bank-logos/"]
 paypal.com
 
 CSS
+input {
```

Comment:
Is there a way to be more specific there (by referencing elements with class names for example).

Reply:
I'm afraid that all the inputs will look different when not expected. Please use the selectors that correspond with website's own border styles.